src/wasm32/ffi.c

```diff
@@ -759,9 +524,8 @@ ffi_prep_closure_loc_js,
   }
   if (nfixedargs < nargs) {
     // extra pointer to varargs stack
-    sig += "i";
+    sig += 'i';
```

Comment:
I guess the original was inconsistent between single and double quotes so it's indeed better to be consistent.